### PR TITLE
[Needs Review] B07902021 簡捷

### PR DESCRIPTION
**Pull request by jay-in-git**
A Pull Request (PR) should explain why we need this PR and how you implement it, so others could understand it quickly.
Since this is just a homework, you do not need to explain why and how. 

**Do NOT change the status to NeedsReview if you do not want to be graded yet.**

Please fill in the following questions as feedback. It is totally optional and would not be graded.

## Is there any suggestion?

<br />

### CODE REVIEW

File: src/students/b07902021.py [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+        student: an Student object
+
+    Hints:
+        * Use Student(parameters1, parameters2 ...) to create an object
+          and assign it to a variable
+        * Use <created object>.<object function> to call object function
+    '''
+    # TODO: create a student object with different words to say
+    student = Student(student_id , time)
+    student.words_to_say = "shit"
```
Comment (amjltc295) [inline src/students/b07902021.py]: Do not swear here.

**Comment by amjltc295** [comment]
@yui1910 Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ to fix it.

File: src/students/b07902021.py (hunk 1 above)
Comment (jay-in-git) [reply src/students/b07902021.py]: @amjltc295  I am sorry for that , and I just fixed it.
Thank you!

**Comment by jay-in-git** [comment]
> @yui1910 Your commit message is not linked to your profile. See https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/ to fix it.

@amjltc295  Excuse me , yui1910 is not my account.  Did you mean that my commit message is linked to his/her account or that this comment was meant to send to him/her?

**Comment by amjltc295** [comment]
@jay-in-git No, I just copy-pasted without changing the name. However, you got the same problem.

**Comment by jay-in-git** [comment]
> @jay-in-git No, I just copy-pasted without changing the name. However, you got the same problem.

@amjltc295  I just fix the problem. Thank you for the solution!!

**Comment by amjltc295** [comment]
@jay-in-git Please include only one file changed.

**Comment by jay-in-git** [comment]
> @jay-in-git Please include only one file changed.

@amjltc295 I just rebased the commit and push a new one. Hope there is no problem now. Thanks!

**Comment by jay-in-git** [comment]
@amjltc295 Hi. Is there still any problem in my PR? Because my friend had just been approved while I am still blocked. Thank you.

**Comment by amjltc295** [comment]
-5 for swearing

**Pull request merged**